server/manager/src/test/resources/conf/accumulo-site.xml

```diff
@@ -82,7 +82,7 @@
   </property>
 
   <property>
-    <name>tserver.walog.max.size</name>
+    <name>tserver.wal.max.size</name>
```

Comment:
Eek. Do we still support XML files in 2.x? I thought we got rid of them. I don't see a reference to this file anywhere. I had thought maybe as a test case somewhere, but I didn't find it. This file might be able to be just deleted outright.

Reply:
Would this be best to do in a future PR?

Reply:
Yeah this file can be removed in a separate PR.